### PR TITLE
Replaces hot/cold loop binary gas pumps with high power pumps

### PR DESCRIPTION
**Pull request by OolongCow**
Replaces the binary pumps for emptying cans into the supermatter's cold and hot loops with high power pumps. The current tickrate makes attempting to use binaries for more than the most basic, mindless of setups a Sisyphean nightmare, and unlike atmos techs, engineers have to toe a fine IC line in order to circumvent a server performance issue.

Screenshot_3

### CODE REVIEW

**Comment by OolongCow** [comment]
!review

File: maps/sccv_horizon/sccv_horizon-2_deck_2.dmm [hunk 1]
```diff
@@ -44806,7 +44806,7 @@
 	input_tag = "cooling_in";
 	name = "Engine Cooling Control";
 	output_tag = "cooling_out";
-	sensors = list("engine_sensor"="Engine Core")
+	sensors = list("engine_sensor"="Engine  Core")
```
Comment (SleepyGemmy) [inline maps/sccv_horizon/sccv_horizon-2_deck_2.dmm:44809]: why?

Comment (OolongCow) [reply maps/sccv_horizon/sccv_horizon-2_deck_2.dmm:44809]: It's some error with SDMM I think. It happens every time I open deck two. Thought I caught this.

Comment (OolongCow) [reply maps/sccv_horizon/sccv_horizon-2_deck_2.dmm]: Fixed now.

**Review by Arrow768 — APPROVED** [review]

**Review by alsoandanswer — APPROVED** [review]

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**